staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config_test.go

```diff
@@ -232,6 +232,13 @@ func TestEncryptionProviderConfigCorrect(t *testing.T) {
 		t.Fatalf("should result in error while parsing configuration file: %s.\nThe file was:\n%s", err, invalidConfigWithAesGcm)
 	}
 
+	invalidConfigWithTypo := "testdata/invalid-configs/invalid-typo.yaml"
+	_, err = LoadEncryptionConfig(ctx, invalidConfigWithTypo, false, "")
+	if got, want := errString(err),
+		`error while parsing file: error decoding encryption provider configuration file "testdata/invalid-configs/invalid-typo.yaml": strict decoding error: unknown field "resources[0].providers[3].kms.pandas"`; got != want {
```

Comment:
can this check for an error containing the bad field name instead of the exact message to make this less sensitive to unimportant tweaks in the formatting?

Reply:
Done.